src/util/decoders.js

```diff
@@ -189,28 +205,34 @@ export function decodeToString(message: Uint8Array): string {
 
 export function asciiToHex(message: string): string {
   var result = [];
-	for (let i = 0; i < message.length; i++) {
-		var hex = Number(message.charCodeAt(i)).toString(16);
-		result.push(hex);
+  for (let i = 0; i < message.length; i++) {
+    var hex = Number(message.charCodeAt(i)).toString(16);
```

Comment:
grumbling at myself: 
```suggestion
    let hex = Number(message.charCodeAt(i)).toString(16);
```

Reply:
Here we can use const I think, each loop is a separate block (scope), so they do not conflict with each other.